src/iocore/cache/CacheDir.cc

```diff
@@ -971,7 +971,7 @@ void
 sync_cache_dir_on_shutdown()
 {
   Dbg(dbg_ctl_cache_dir_sync, "sync started");
-  EThread *t = (EThread *)0xdeadbeef;
```

Comment:
What about `EThread *t{0xdeadbeef}`? Can it be initialized directly from the value?

Reply:
`clang++` doesn't like it 😢 
```
src/iocore/cache/CacheDir.cc:974:14: error: cannot initialize a variable of type 'EThread *' with an rvalue of type 'unsigned int'
  974 |   EThread *t{0xdeadbeef};
      |              ^~~~~~~~~~
1 error generated.
```

Reply:
Ah well. A lot of the casts are sadly necessary.